.github/workflows/zcashd-cd.yml

```diff
@@ -0,0 +1,77 @@
+name: Zcashd Manual Deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        default: 'testnet'
+      size:
+        default: 10
+
+env:
+  PROJECT_ID: zealous-zebra
+  REGION: us-east1
+  ZONE: us-east1-b
+
+jobs:
+
+  deploy:
+    name: Deploy mainnet nodes
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set project and image names
+        run: |
+          BRANCH_NAME=$(expr $GITHUB_REF : '.*/\(.*\)') && \
+          BRANCH_NAME=${BRANCH_NAME,,} && \
+          REPOSITORY=${GITHUB_REPOSITORY,,} && \
+          echo "BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_ENV && \
+          echo "SHORT_SHA=$(git rev-parse --short=7 $GITHUB_SHA)" >> $GITHUB_ENV && \
+          echo "REPOSITORY=$REPOSITORY" >> $GITHUB_ENV
+
+      # Setup gcloud CLI
+      - name: Set up gcloud SDK environment
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '295.0.0'
+          project_id: ${{ env.PROJECT_ID }}
+          service_account_key: ${{ secrets.GCLOUD_AUTH }}
+
+      # Create instance template from container image
+      - name: Create instance template
+        run: |
+          gcloud compute instance-templates create-with-container "zcashd-$BRANCH_NAME-$SHORT_SHA" \
+          --container-image "electriccoinco/zcashd" \
+          --container-env ZCASHD_NETWORK="${{ github.event.inputs.network }}" \
+          --machine-type n2d-standard-4 \
+          --service-account cos-vm@zealous-zebra.iam.gserviceaccount.com \
+          --scopes cloud-platform \
+          --tags zcashd \
```

Comment:
I just updated the firewall rule to allow instances with this tag to also allow incoming traffic on the 8233, 18233 ports, same as the `zebrad` tag